ruby/lib/rspec/queue/build_status_recorder.rb

```diff
@@ -3,6 +3,8 @@ module Queue
     class BuildStatusRecorder
       ::RSpec::Core::Formatters.register self, :example_passed, :example_failed
 
+      COUNTERS = ['error-reports']
```

Comment:
Not sure this constant is necessary.

Reply:
Turns out there's no need to reset counters at all with rspec, so I added a test and removed the `reset_counters` method entirely

Reply:
Because the try deletes the original key on success: https://github.com/Shopify/ci-queue/blob/master/ruby/lib/ci/queue/redis/build_record.rb#L37